src/boot/efi/boot.c

```diff
@@ -87,6 +97,7 @@ typedef struct {
         BOOLEAN auto_entries;
         BOOLEAN auto_firmware;
         BOOLEAN reboot_for_bitlocker;
+        enum secure_boot_enrollment sb_enrollment;
```

Comment:
don't abbreviate this needless please, i.e. sb → secure_boot

we are fine with using short variable names as long as they have local scope. But in shared codes, plesae don't abbreviate needlessly.